gamedev-iitk/tower-defense
# Adding a Dynamic NavMesh

**Description:** Dynamic NavMesh created for towers


**Steps to test:**
1.Upgrade the tower, and move them
2. Check out the Navigator window
3. You will see the navmesh changes when the tower moves.
**Outstanding:**
1. Changing the size of the navmesh obstacle under the tower prefabs
